### PR TITLE
Purge "multinomial"

### PR DESCRIPTION
**Pull request by alldefector**

### CODE REVIEW

File: compiler/compile-config/compile-config-2.01-grounding [hunk 1]
```diff
@@ -378,15 +378,15 @@ def factorWeightDescriptionSqlExpr:
                 # Not using DISTINCT: User decides if they want to have duplicate factors
                 { SELECT:
                     [ ( .function_.variables[] | { table: "F", column: .columnId } )
-                    # TODO XXXX: assuming so-called "multinomial" factors' heads have categorical vars only
-                    , ( .function_ | select(.isMultinomial) | .variables[] |
+                    # TODO XXXX: assuming categorical factors' heads have categorical vars only
```
Comment (netj) [inline compiler/compile-config/compile-config-2.01-grounding]: I think three-X is the convention: `XXX`.  Editors and IDEs recognize them and highlight just like `TODO` and `FIXME`.  Not sure where they came from but pretty useful to flag comments.

**Comment by netj** [comment]
Looks great!  Should we also get rid of the categories column in the weights view here, I remember you flagged as unnecessary in a previous commit?

File: compiler/compile-config/compile-config-2.01-grounding (hunk 1 above)
Comment (alldefector) [inline compiler/compile-config/compile-config-2.01-grounding]: I thought more Xs could get more attention :)

**Comment by alldefector** [comment]
I assume you meant dd_graph_weights.categories? Removing now.

**Pull request merged**